src/client/src/api/fetchApiV2.js

```diff
@@ -142,3 +154,65 @@ export const useFaciesDescription = selectedStratigraphyID =>
     queryKey: [faciesDescriptionQueryKey, selectedStratigraphyID],
     queryFn: () => fetchFaciesDescriptionsByProfileId(selectedStratigraphyID),
   });
+
+export const chronostratigraphiesQueryKey = "chronostratigraphies";
+
+export const useChronostratigraphies = stratigraphyID =>
```

Comment:
Vielleicht wäre es auch noch etwas fürs Backlog. Einfach mal festhalten, dass wir den ganzen Gebrauch von UseQuery in der App mal vereinheitlichen und die Queries dann wo immer möglich einsetzen um möglichst wenig Zweigleisigkeiten zu haben.